### PR TITLE
Israel/0.0.37 instructions

### PR DESCRIPTION
**Pull request by israel-resonai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,59 +3,64 @@
...
+Then, sync project:
 
 ## Usage 
-  Create a configuration object and start sdk  for both case 
+  Create a configuration object and start sdk for both case
 
-
-With Login :
-    To track user information you can  pass user data.
+```groovy
```
Comment (shirl-resonai) [inline README.md]: This isn't groovy - it's Java

File: README.md [hunk 2]
```diff
@@ -3,59 +3,64 @@
...
+VeraConfiguration.Builder veraBuilder = new VeraConfiguration.Builder
+        // (this) Activity used
+        (getSupportFragmentManager(), R.id.container) // pass container for fragment
+        .setClientAppID("vera_client_app")) // Your custom Vera platform ID.
+        .setSiteIDs("siteId1", "siteId2");
+        .setShowCloseButton(shouldShow) // A boolean value of whether the SDK should show it's own close button - default to true
```
Comment (shirl-resonai) [inline README.md]: "its" (no apostrophe)

File: README.md [hunk 3]
```diff
@@ -3,59 +3,64 @@
...
+        .setHideHeader(false) // A boolean value for the SDK to hide or show the default header. Defaults to `false`
+        .setDeeplinkPrefix(veraDomain) //A string value defining your custom "deepLinkPrefix". Used when generating deep links that you can pass into Vera and should open your app.
+        .setLanguage(sdkLanguage) // Choose vera language
+        .onRequestRefreshToken(this::fetchUserToken) // Listener triggered when user token need refresh
+        .onMessageListener(this::onMessage) // Listener to messages from vera and implement logic for them
+        .onCloseListener(this::closeVeraSdk) // Listener triggered when vera closed
+```
+You can Start vera with user data :
```
Comment (shirl-resonai) [inline README.md]: I would write "start" (lower case)

File: README.md [hunk 4]
```diff
@@ -3,59 +3,64 @@
...
+        (getSupportFragmentManager(), R.id.container) // pass container for fragment
+        .setClientAppID("vera_client_app")) // Your custom Vera platform ID.
+        .setSiteIDs("siteId1", "siteId2");
+        .setShowCloseButton(shouldShow) // A boolean value of whether the SDK should show it's own close button - default to true
+        .setHideHeader(false) // A boolean value for the SDK to hide or show the default header. Defaults to `false`
+        .setDeeplinkPrefix(veraDomain) //A string value defining your custom "deepLinkPrefix". Used when generating deep links that you can pass into Vera and should open your app.
+        .setLanguage(sdkLanguage) // Choose vera language
+        .onRequestRefreshToken(this::fetchUserToken) // Listener triggered when user token need refresh
+        .onMessageListener(this::onMessage) // Listener to messages from vera and implement logic for them
+        .onCloseListener(this::closeVeraSdk) // Listener triggered when vera closed
```
Comment (shirl-resonai) [inline README.md]: It's minor but I would pick a single style for all lines' comments. For instance:
`// Something like this for short comments`  (starting with space and uppercase, ends without period)
``// A longer comment can look like this with a period. Defaults to `somevalue` ``

File: README.md [hunk 5]
```diff
@@ -3,59 +3,64 @@
...
+```
+You can Start vera with user data :
 
 ```groovy
```
Comment (shirl-resonai) [inline README.md]: also not groovy (I guess this was here before) - same throughout the file

File: README.md [hunk 6]
```diff
@@ -3,59 +3,64 @@
...
 
-        VeraConfiguration.Builder()
-               .setClientAppID("vera_client_app")
-               .setLanguage(Languages.EN)
-               .startWithLogin(this, "userName", "userId", "token");
+     builder.startWithLogin(useName(), userId, token);
 ```
-Without login as :
+
+You can also start vera without user data :
```
Comment (shirl-resonai) [inline README.md]: I would write that this is an anonymous login, such as:

`You can also start vera without user data, i.e. anonymously:`

File: README.md [hunk 7]
```diff
@@ -3,59 +3,64 @@
...
-                  .setClientAppID("vera_client_app")
-                  .setLanguage(Languages.EN)
-                  .startWithoutLogin(this);
+
+     builder.startWithoutLogin();
 ```
 
+```groovy
+
+    when you need to send vera data after vera already exist you can use setDeeplinkComponent:
```
Comment (shirl-resonai) [inline README.md]: * make "Vera" capitalized (other comments in this file too if there are any)
* capitalize "When"
* "after Vera already exist**s**, you can" etc. - is it clear what you mean by "exists" as opposed to say "is running" ?

Comment (shirl-resonai) [reply README.md]: * backtick `` `setDeelinkComponent` ``

File: README.md (hunk 4 above)
Comment (israel-resonai) [reply README.md]: I tried to make it short as I can, and added default if exist

**Review by shirl-resonai — APPROVED** [review]

**Pull request merged**